packages/code-du-travail-frontend/src/outils/DureePreavisLicenciement/steps/__tests__/Status.test.js

```diff
@@ -0,0 +1,9 @@
+import { StepStatus } from "../Status";
+import { renderForm } from "../../../../../test/renderForm";
+
+describe("<StepStatus />", () => {
+  it("should render", () => {
```

Comment:
Since there is logic in this step you should also test that logic  (serious misconduct / disabledWorker)

Reply:
OK